### DISCUSSION TITLE
Welcome to VoyagerTelegramBot Discussions!

### DISCUSSION THREAD

**Discussion by sly9** [sly9/VoyagerTelegramBot#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gavinsboyle** [comment]
Hi guys
Gavin here.
good to see the discussion group restarting here.
can see why its tricky on the main voyager forum and this is a good solution for all.
will start to give suggestions soon...
best
gavin
